src/automerge.js

```diff
@@ -27,14 +28,20 @@ function from(initialState, options) {
 
 function change(doc, options, callback) {
   const [newDoc, change] = Frontend.change(doc, options, callback)
+  newDoc[LOCAL] = change
```

Comment:
I think it would be neater to do this in the `makeChange` function in `frontend/index.js`